python/hail/keytable.py

```diff
@@ -1307,28 +1307,29 @@ def union(self, *kts):
         return KeyTable(self.hc, self._jkt.union([kt._jkt for kt in kts]))
 
     @handle_py4j
+    @record_method
     @typecheck_method(n=integral)
-    def take(self, n):
+    def head(self, n=10):
         """Take a given number of rows from the head of the table.
-        
+
         **Examples**
-        
-        Take the first ten rows:
-        
-        >>> first10 = kt1.take(10)
-        
+
+        Take the first 50 rows:
```

Comment:
This is also confusing me. It could be because `take` means something specific in Spark, and this is doing something different.